.changeset/config.json

```
@@ -1,7 +1,7 @@
 {
   "$schema": "https://unpkg.com/@changesets/config@2.3.1/schema.json",
   "commit": false,
-  "changelog": false,
+  "changelog": false,  
```

```suggestion
  "changelog": false,
```
We might be missing a code formatter in the repo to enforce this

---

yep, it wasn't targeting the file, fixed.